website/lib/build.js

```diff
@@ -16,11 +16,20 @@ fs.copySync('../dist/katex.min.js', 'static/static/katex.min.js');
 fs.copySync('../dist/katex.min.css', 'static/static/katex.min.css');
 fs.copySync('../dist/fonts', 'static/static/fonts');
 
+// copy local built KaTeX extensions
+fs.copySync('../dist/contrib/copy-tex.min.css', 'static/static/copy-tex.min.css');
+fs.copySync('../dist/contrib/copy-tex.min.js', 'static/static/copy-tex.min.js');
+fs.copySync('../dist/contrib/mhchem.min.js', 'static/static/mhchem.min.js');
+
+
 // use KaTeX from CDN on the main page for Netlify production deploy
 if (process.env.CONTEXT === 'production') {
     const version = require('../versions.json')[0];
     let indexHtml = fs.readFileSync('pages/index.html', 'utf8');
     indexHtml = indexHtml.replace(/(["'])static\/(katex|fonts)/g,
         `$1https://cdn.jsdelivr.net/npm/katex@${version}/dist/$2`);
+        //CDN pathway different for `copy-tex` and `mhchem`
```

Comment:
very very minor: You could chain the replaces like this:
```suggestion
        `$1https://cdn.jsdelivr.net/npm/katex@${version}/dist/$2`)
         .replace(/(["'])static\/(copy-tex|mhchem)/g,
            `$1https://cdn.jsdelivr.net/npm/katex@${version}/dist/contrib/$2`);
```

Reply:
I did think about chaining it. But I realized it might more readable if I break it up because each replace is quite long. Any thoughts?

Reply:
I'm okay either way.